backend/app/tests/core/test_security.py

```
@@ -0,0 +1,74 @@
+    """Test successful decryption of a password."""
+    plain_password = "testpassword123"
+    
+    # Encrypt the password using the test key
+    cipher_suite = Fernet(TEST_FERNET_KEY)
+    encrypted_password_bytes = cipher_suite.encrypt(plain_password.encode('utf-8'))
+    encrypted_password_b64 = encrypted_password_bytes.decode('utf-8')
+
+    decrypted_password = decrypt_password(encrypted_password_b64)
+    assert decrypted_password == plain_password
+
+def test_decrypt_password_invalid_token(mock_settings_key):
```

Ruff (0.11.9)

Unused function argument: `mock_settings_key`

(ARG001)